torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13441,6 +13498,9 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                 toleranceOverride({torch.bfloat16: tol(atol=1e-02, rtol=1.6e-02)}), 'TestUnaryUfuncs',),
             DecorateInfo(toleranceOverride({torch.float16: tol(atol=1e-02, rtol=1e-02)}),
                          'TestCudaFuserOpInfo', 'test_nvfuser_correctness'),
+            DecorateInfo(toleranceOverride({torch.complex64: tol(atol=6e-04, rtol=1e-05)}),
+                         'TestUnaryUfuncs', 'test_reference_numerics_extremal',
+                         device_type='cuda'),
```

Comment:
The bfloat16 failure is due to this `toleranceOverride` taking precedence over the one above for bfloat16. I can get the test to pass locally by adding copying bfloat16's `tol` term from above here.